src/Module/Admin/Site.php

```diff
@@ -698,7 +698,7 @@ public static function content(array $parameters = [])
 			'$worker_frontend'        => ['worker_frontend', DI::l10n()->t('Enable frontend worker'), DI::config()->get('system', 'frontend_worker'), DI::l10n()->t('When enabled the Worker process is triggered when backend access is performed (e.g. messages being delivered). On smaller sites you might want to call %s/worker on a regular basis via an external cron job. You should only enable this option if you cannot utilize cron/scheduled jobs on your server.', DI::baseUrl()->get())],
 
 			'$relay_subscribe'        => ['relay_subscribe', DI::l10n()->t('Subscribe to relay'), DI::config()->get('system', 'relay_subscribe'), DI::l10n()->t('Enables the receiving of public posts from the relay. They will be included in the search, subscribed tags and on the global community page.')],
-			'$relay_server'           => ['relay_server', DI::l10n()->t('Relay server'), DI::config()->get('system', 'relay_server', 'https://relay.diasp.org'), DI::l10n()->t('Address of the relay server where public posts should be send to. For example https://relay.diasp.org')],
+			'$relay_server'           => ['relay_server', DI::l10n()->t('Relay server'), DI::config()->get('system', 'relay_server', 'https://relay.diasp.org'), DI::l10n()->t('Address of the relay server where public posts should be send to. For example %s', 'https://social-relay.isurf.ca')],
```

Comment:
Should the default value for the `DI::config()->get` call be the server out of order or none at all?

Reply:
Is changed to the new one.

Reply:
But should there be a default relay at all? What if I don't want to use the relay, will the set URL be ignored?

Reply:
Since this is no new configuration, I guess it had worked before to just leave that field empty if you don't want to participate.